pyproject.toml

```
@@ -19,7 +19,7 @@ mock = "^4.0.3"
 protobuf = "^3.19.3"
 protoc-gen-validate = "^0.4.1"
 python = "^3.7"
-typing-extensions = "^3.7.4"
+typing-extensions = ">3.7.4,<5"
```

```suggestion
typing-extensions = ">=3.7.4,<5"
```
Just in case there are users depending on 3.7.4

---

Oops, thanks! I've pushed a new commit with that fixe this